src/joyride/repl_utils.cljs

```diff
@@ -28,6 +28,25 @@
               (when (re-find pat (str sym-ns "/" sym-name))
                 [sym-ns (str sym-ns "/" sym-name)]))))))
 
+(defn- instance-completions [instance prefix]
+  (let [props (loop [obj instance
+                     props []]
+                (if obj
+                  (recur (js/Object.getPrototypeOf obj)
+                         (into props (js/Object.getOwnPropertyNames obj)))
+                  props))
+        completions (map (fn [k]
+                           [nil (str prefix k)]) props)]
+    completions))
+
+(defn instance-properties
+  "Returns properties of `instance`, attempting to correspond with what `node:repl.completer` would return."
+  [instance]
+  (->> (instance-completions instance nil)
+       (map second)
+       (remove #{"__lookupGetter__" "__defineSetter__"
```

Comment:
Would it be an idea to complete skip everything that starts with `__`?

Reply:
I considered, but since `completer` keeps `__proto__`, I thought it could have reasons to. But, yeah, maybe it is a bit arbitrary... Please make the call. 😄

Reply:
Or just leave those things in, always?